packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -55,6 +56,12 @@ class ListWheelParentData extends ContainerBoxParentData<RenderBox> {
   ///
   /// This must be maintained by the [ListWheelChildManager].
   int? index;
+
+  /// Transform applied to this child during painting.
+  ///
+  /// Can be used to find the local bounds of this child in the viewport,
+  /// and then use it, for example, in hit testing.
+  Matrix4? transform;
```

Comment:
For the docs, when would this be null?

Reply:
From what it seems, never. Added docs and assertions to ensure that.